src/propagate_constant.cpp

```diff
@@ -83,7 +84,12 @@ void propagate_constant::apply(module& m) const
     // Compute literals in parallel
     std::vector<instruction_ref> const_instrs_vec{const_instrs.begin(), const_instrs.end()};
     std::vector<argument> literals(const_instrs_vec.size());
-    simple_par_for(const_instrs_vec.size(), 1, [&](const auto i) {
+    int n = 1;
+#ifndef ParallelSTL_FOUND
```

Comment:
You should do `#if !MIGRAPHX_HAS_EXECUTORS`. I dont think `ParallelSTL_FOUND` is ever defined.